### ISSUE TITLE
→ Accessibility: Components | Switch

### ISSUE THREAD

**Issue by yceballost** [Telefonica/mistica-design#1492]
#557 

aria-hidden="true" should be used in on and off spans

https://www.w3.org/WAI/ARIA/apg/example-index/switch/switch.html

Current DOM structure:
```

<span role="switch" aria-checked="true" tabindex="0" aria-labelledby="uncontrolled-label">
  {...
      <span>on</span>
   }
</span>
```
Proposed DOM structure:

```
<span role="switch" aria-checked="true" tabindex="0" aria-labelledby="uncontrolled-label">
  {...
      <span aria-hidden="true">on</span>
   }
</span>
```

## Tasks

- [x] Update specs
- [ ] Design documentation
- [x] Android [ticket](https://jira.tid.es/browse/ANDROID-13948)
- [x] iOS [ticket](https://jira.tid.es/browse/IOS-9506)
- [x] Web [ticket](https://jira.tid.es/browse/WEB-1624)

**Comment by yceballost** [comment]
Should we include this visual labels?
![image](https://github.com/Telefonica/mistica-design/assets/6722153/cff76e68-acf7-489b-b40a-6f14ab194f4d)
